commune_app/models.py

```diff
@@ -1,3 +1,8 @@
-# from django.db import models
+from django.db import models
 
-# Create your models here.
+class Chore(models.Model):
+    #commune=models.ForeignKey(commune,on_delete=models.CASCADE)
+    title=models.CharField(max_length=50)
```

Comment:
change title=models to title = models
same for date, assign_to and description

Reply:
fixed it